src/esp/sim/Simulator.h

```diff
@@ -297,6 +312,22 @@ class Simulator {
                   const int objectID,
                   const int sceneID = 0);
 
+  /**
+   * @brief Apply an impulse to an object. See @ref
+   * esp::physics::PhysicsManager::applyImpulse.
+   * @param impulse The desired linear impulse to apply.
+   * @param relPos The desired location relative to the object origin at which
+   * to apply the impulse.
+   * @param objectID The ID of the object identifying it in @ref
+   * esp::physics::PhysicsManager::existingObjects_.
+   * @param sceneID !! Not used currently !! Specifies which physical scene of
+   * the object.
+   */
+  void applyImpulse(const Magnum::Vector3& impulse,
+                    const Magnum::Vector3& relPos,
+                    const int objectID,
+                    const int sceneID = 0);
```

Comment:
```suggestion
                    int objectID,
                    int sceneID = 0);
```

in the declaration it's just noise :) Same above (and elsewhere) I think?

Reply:
@aclegg3 just to clarify after seeing the diff -- I meant just the declaration, it could stay as-is in the definition (`const` is good, after all). 

However if your IDE (or some linter) forces you to have definition and declaration match, then you can't do much :/ (I hope that's not the case?)